src/com/model/BanknotesCounter.java

```diff
@@ -0,0 +1,31 @@
+package com.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BanknotesCounter {
+    // Allowed banknotes to be deposited;
+    public final int[] ALLOWED_BANKNOTES = {1, 3, 5, 10, 20, 100};
```

Comment:
If this is a constant (probably it is), then, please declare it as `public static final int[]...`